lms/djangoapps/courseware/access.py

```diff
@@ -878,12 +878,18 @@ def get_user_role(user, course_key):
     """
     role = get_masquerade_role(user, course_key)
 
-
     # Check if user is a coach on this ccx.
-    ccx_coach_role = False # Added by labster
+    ccx_coach_role = False  # Added by labster
     if isinstance(course_key, CCXLocator):
         try:
-            ccx_coach_role = has_ccx_coach_role(user, course_key)
+            ccx = CustomCourseForEdX.objects.get(pk=course_key.ccx)
```

Comment:
yes it fix. 
the difference is the course key that send for check.

https://github.com/Livit/Livit.Learn.EdX/blob/e4a2e5d7cc9054f40d261dd11fde46b60b9137c3/lms/djangoapps/courseware/access.py#L86-L89
and
https://github.com/Livit/Livit.Learn.EdX/blob/e4a2e5d7cc9054f40d261dd11fde46b60b9137c3/lms/djangoapps/courseware/access.py#L884-L885